Bandit Security Check

Lcjacobpd opened the pull request:
Added bandit security test to qa/
Updated run-qa-suit to include new test
Added # nosec to key lines in public/host.py to pass security
checks

Lcjacobpd committed:
Bandit

Added bandit security test to qa/
Updated run-qa-suit to include new test
Added # nosec to key lines in public/host.py to pass security
checks

samuelkgutierrez commented on qa/run-bandit-tests:
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Copyright (c)      2020 Triad National Security, LLC
```
Please fix Copyright dates

samuelkgutierrez commented on qa/run-bandit-tests:
```diff
@@ -0,0 +1,16 @@
+
+echo "Reporting on Security Standing"
+
+# pip install bandit
```
You need to incorporate this dependency in the proper place for GitHub actions; remove this comment. Further, this needs to be added to the suite of tests that are called individually in our CI QA suite.

samuelkgutierrez reviewed (changes requested):
Please review and address the issues raised. Thank you.

Lcjacobpd committed:
Bandit GitHub Actions

Corrected copyright date in qa/run-bandit-tests
Added pip install bandit check to qa/run-packaging-tests
Removed comment qa/run-bandit-tests
Added bandit tests to .github/workflows/qu.yml

Lcjacobpd committed:
Packaging Test Revert

Removed call to pip install bandit in qa/run-packaging-test
Corrected copyright date to include start-current

Lcjacobpd committed:
Copyright date

Updated copyright date in remaining modified files

samuelkgutierrez commented on qa/run-bandit-tests:
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c)      2020 Triad National Security, LLC
+# Copyright (c)      2021 Triad National Security, LLC
```
This should read StartYear-CurrentYear for all modified files.

samuelkgutierrez commented on qa/run-packaging-tests:
```diff
@@ -12,3 +12,5 @@
 echo "Testing Packaging"
 
 python3 -m pip install --user .
+
```
Why did you add this?

samuelkgutierrez reviewed (changes requested):
Please address issues raised in comments.

samuelkgutierrez reviewed (approved):
Looks fine. Thank you.
